### PR TITLE
Remove removable utility libraries

### PR DESCRIPTION
**Pull request by rootEnginear**
Hi! This PR is to reduce the extension size by removing removable utility libraries. The removed libraries are jQuery, Bootstrap JavaScript, and FontAwesome.

## jQuery
jQuery is replaceable with native JavaScript. However, I have to do some workaround with the alert. I have re-coded it to have a better user experience, except for its placement (that's moving layout around) which we might have to discuss that later.

## Bootstrap JavaScript
We didn't use any of the Bootstrap components that required JavaScript. So we can safely remove it 👌.

## FontAwesome
**You use only 2 icons in the extension.** But the font files are including all of the icons, making the extension strangely bloat. The fonts folder itself is ~2.81 MB! So I removed all of the FontAwesome modules entirely and used the official FontAwesome SVGs as a substitute.

## Result?
I have compared the before/after size of them by compressed them into zip files. The sizes are the following:
- **Before:** 2.54 MB
- **After:** 1.18 MB (-46.46%)

## Further improvement?
We can keep on compressing our files by using preprocessors like minifier and uglifier. It comes with an ease of development downside, and I don't think it will be much more significant than now.

If you have any questions after trying it out, please feel free to ask me 👍.

### CODE REVIEW

**Review by shuhaoli — COMMENTED** [review]
Thanks for the work!

**Comment by rootEnginear** [comment]
Adjusted from `1s` to `0.5s`, please see if it's alright 😉.

File: css/popup.css [hunk 1]
```diff
@@ -36,3 +36,49 @@
...
+        opacity: 0;
+        margin: 0;
+        height: 0;
+        font-size: 0;
+        border: 0;
+    }
+}
+
+.alert {
+    animation: alertInAnimation 1s forwards;
```
[the remark this answers is not in the document]
Comment (rootEnginear) [reply css/popup.css]: Adjusted from 1s to 0.5s, please see if it's alright 😉.

**Comment by shuhaoli** [comment]
Thanks again for the work!

**Pull request merged**